app/src/main/java/org/oppia/app/player/audio/AudioViewModel.kt

```diff
@@ -14,7 +14,7 @@ import org.oppia.domain.audio.AudioPlayerController.PlayProgress
 import org.oppia.domain.audio.AudioPlayerController.PlayStatus
 import org.oppia.util.data.AsyncResult
 import org.oppia.util.gcsresource.DefaultResource
-import java.util.Locale
+import java.util.*
```

Comment:
Please fix--we shouldn't be including wildcard imports per https://developer.android.com/kotlin/style-guide#import_statements.

Reply:
Done

Reply:
Checked and its correct. (Resolving on behalf of Ben)